indico/modules/events/surveys/templates/emails/start_notification_email.txt

```diff
@@ -0,0 +1,11 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    {% trans %}Survey has started{% endtrans %}
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    Survey "{{ survey.title }}" is now open. From now on, users can submit the answers to the survey.
```

Comment:
"The survey..". Also, users don't know about surveys until they are open and this is an email to users, so talking generally about "users" doesn't fit well here.

Actually.. if wonder if we should add a field for a custom message to send here.

Also, this email should include a link to the survey - users won't lookup the event (the email doesn't even mention which event it is) just to go to a survey. They expect a direct link to get there...